usaspending_api/broker/v2/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.conf.urls import url
+from rest_framework.routers import DefaultRouter
+
+from usaspending_api.broker.v2 import views
+
+broker_router = DefaultRouter()
+broker_router.register('broker', views.DetachedAwardProcurementViewSet)
+broker_router.register('broker', views.PublishedAwardFinancialAssistanceViewSet)
+
+mode_list = {'get': 'list', 'post': 'list'}
+mode_detail = {'get': 'retrieve', 'post': 'retrieve'}
+
+
+urlpatterns = [
+    url(r'^detachedawardprocurementviewset/$', views.DetachedAwardProcurementViewSet.as_view({'get': 'list'})),
```

Comment:
Probably not - I wanted to throw something up there for visibility, but we can drop views.py, serializer.py, and both urls.py files from this branch and just have the models, settings.py, and migrations.